pkg/skaffold/build/sequence_test.go

```diff
@@ -78,9 +137,48 @@ func TestInSequence(t *testing.T) {
 				{ImageName: "skaffold/image2"},
 			}
 
-			got, err := InSequence(context.Background(), out, test.tags, artifacts, test.buildArtifact)
+			cfg := latest.BuildConfig{
+				BuildType: latest.BuildType{
+					LocalBuild: &latest.LocalBuild{},
+				},
+			}
+			event.InitializeState(&runcontext.RunContext{
+				Cfg: &latest.Pipeline{
+					Build: cfg,
+				},
+				Opts: &config.SkaffoldOptions{},
+			})
+
+			buildResultChannels, err := InSequence(context.Background(), out, test.tags, artifacts, test.buildArtifact)
+			testutil.CheckError(t, test.shouldErr, err)
+
+			res := CollectResultsFromChannels(buildResultChannels)
+
+			fmt.Fprintf(os.Stdout, "final build results: %+v\n", res)
+
+			// build results are returned in a list, of which we can't guarantee order.
+			// loop through the expected results, and find the matching build result by target artifact.
+			found := false
+			for _, testRes := range test.expectedResults {
```

Comment:
it looks like there is an opportunity to pull this out into a utility method - it is a straight duplication across parallel_test and sequence_test